src/edu/csus/ecs/pc2/shadow/ShadowController.java

```diff
@@ -95,14 +95,18 @@ public String getLabel() {
             return this.label;
         }
     }
-
+    public enum FILTERS {
+        NONE,
+        ONLY_MISSMATCH
```

Comment:
Check spelling of "Mismatch" throughout the changed files.  Spelled wrong in a couple of places, including the checkbox text.